lgc/builder/BuilderImpl.cpp

```diff
@@ -323,6 +323,29 @@ BranchInst *BuilderImplBase::createIf(Value *condition, bool wantElse, const Twi
   return branch;
 }
 
+// Permissively test two instructions are the same.
```

Comment:
Can you give an example of this permissiveness? For example, two instructions that are identical here but don't return true in `lhs->isIdenticalTo(rhs)`.

> Permissively test two instructions are the same.

nit 1: test **if** / **whether** two...?
nit 2: the function name uses 'identical' but the comment talks about instructions being 'same'. Do 'same' and 'identical' have the same meaning here? If yes, could we just stick to one of these terms so that we don't introduce ambiguity?

Reply:
I have updated the comment to be more descriptive about the point of this function.